.claude/skills/extract-kernel-definitions/SKILL.md

```
@@ -465,20 +466,22 @@ When extracting kernel definitions, use different sources for different purposes
   ```
 - **Pattern**: Look for functions like `ref_attention()`, `ref_rmsnorm()`, `ref_mla()`, etc.
 - **Use for**: Writing the `reference` field in Definition JSON
+- **IMPORTANT**: Always search FlashInfer tests directory FIRST before falling back to SGLang
 
-### For Reference `run()` Implementation: SGLang Vanilla (Fallback Only)
+### For Reference `run()` Implementation: SGLang Vanilla (FALLBACK ONLY)
 - **When**: FlashInfer does NOT have the kernel (e.g., some MoE variants, custom ops)
-- **Location**: `third_party/sglang/python/sglang/srt/layers/`
+- **Location**: `tmp/sglang/python/sglang/srt/layers/`
 - **Examples**:
   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)